src/plugins/DevTools/DevTools.ts

```diff
@@ -3,9 +3,6 @@ import { JWOwlUIPlugin } from '../../ui/JWOwlUIPlugin';
 import devtoolsTemplates from './DevTools.xml';
 
 export class DevTools extends JWOwlUIPlugin {
-    constructor() {
-        super();
-        this.componentsRegistry.push(DevToolsComponent);
-        this.templates = devtoolsTemplates;
-    }
+    componentsRegistry = this.componentsRegistry.concat(DevToolsComponent);
+    templates = devtoolsTemplates;
```

Comment:
This should not be in this commit. It won't not truly fit in any other commit however, since its some basic refactoring, so I suggest you to put it in the commit where you add the registry inspector in DevTools, at least the commit will target DevTools specifically so it will make more sense IMO.